language/sygus.tex

```diff
@@ -315,6 +315,8 @@ \subsubsection{Changes from SyGuS 2.0 to SyGuS 2.1}
 \item A new feature $\fweightskwd$ has been added, which allows weight annotations on terms, 
 as well as the use of weight symbols within terms, as described in \cref{sec:weight-semantics}.
 
+\item A new command $\optsynthkwd$ has been added for specifying optimization queries.
```

Comment:
Unrelated to this PR, but I just realized that we haven't mentioned the theory of tables in this list of changes. Please add it.